include/v8-instance.h

```diff
@@ -13,7 +15,12 @@ class v8_instance {
 public:
     v8_instance(v8::Isolate::CreateParams create_params_)
     : create_params(std::move(create_params_)),
-      isolate(v8::Isolate::New(create_params)) {}
+      isolate(v8::Isolate::New(create_params)) {
+            watchdog.set_callback([this]{
+                stop_execution_loop();
+                is_cancel = true;
+            });
+      }
 
     ~v8_instance() {
         context.Reset();
```

Comment:
v8_instance manages several resources here, I'd suggest to use `unique_ptr` with array buffer allocator here (explicit delete is meh, what if something will throw exception in d-tor?)

Reply:
Done